webapp/src/main/java/com/box/l10n/mojito/smartling/SmartlingAuthorizationCodeAccessTokenProvider.java

```diff
@@ -50,18 +50,24 @@ public OAuth2AccessToken obtainAccessToken(OAuth2ProtectedResourceDetails detail
         request.put("userIdentifier", details.getClientId());
         request.put("userSecret", details.getClientSecret());
 
-        DefaultOAuth2AccessToken defaultOAuth2AccessToken = null;
-        try {
-            DateTime now = getNowForToken();
-            AuthenticationResponse authenticationResponse = restTemplate.postForObject(details.getAccessTokenUri(), request, AuthenticationResponse.class);
-            defaultOAuth2AccessToken = getDefaultOAuth2AccessToken(now, authenticationResponse);
-        } catch (Exception e) {
-            String msg = "Can't get Smartling token";
-            logger.debug(msg, e);
-            throw new OAuth2AccessDeniedException(msg, details, e);
+        OAuth2AccessToken accessToken = null;
+        OAuth2AccessToken existingToken = accessTokenRequest.getExistingToken();
+        if(existingToken != null && existingToken.getRefreshToken() != null){
```

Comment:
knit: can you re-indent the "if" statement? we don't have auto formatting but the default from Intellij should be a good first step.